templates/taxi/car_detail.html

```diff
@@ -9,4 +9,6 @@ <h1>Drivers</h1>
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
   </ul>
+  <a class="btn btn-primary" href="{% url 'taxi:car-update' car.id %}">Update</a>
```

Comment:
The URL tag uses `car.id`, but it should be `car.pk` to ensure consistency with Django's convention for primary keys. Consider changing `car.id` to `car.pk`.